### DISCUSSION TITLE
TPA clickable Accept and Decline

### DISCUSSION THREAD

**Discussion by VinZar-gg** [EssentialsX/Essentials#5432] [Ideas and Feature Suggestions] [answered]
Any TPA requests require the other person to type out /tpaccept or /tpdeny which can be a hassle. I have tried the TPA Datapack from Vanilla Tweaks, and I liked the feature where you can click on tpaccept and tpdeny on the chat interface and it will execute that command. I think it would be a very useful and wanted feature.

**Comment by LaxWasHere** [comment, marked as answer]
This is already possible with the upcoming adventure update.

**Reply by VinZar-gg** [reply]
When is the update coming out?

**Reply by l4n-socrates** [reply]
still not possible, not in stable and not in dev build. Are there any new informations about this request?

**Reply by Evidentsinger14** [reply]
When there is any progress on it, it will be made known on #4717 - the PR is still waiting for review completion, and docs to be finished for it

**Comment by Evidentsinger14** [comment]
Adventure has been merged, and this is now possible. check out the docs on Adventure/MiniMessage with click components.
